### PR TITLE
fix(components): Padding Update to RadioGroup Component

### PR DESCRIPTION
**Pull request by tyler-santos**
## Motivations

Small update to add more padding between each radio button option.

## Changes

#Before
Screen Shot 2020-05-07 at 2 19 13 PM

#After
Screen Shot 2020-05-07 at 2 20 23 PM

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by tyler-santos** [comment]
@seanhealy Hey Sean, super small PR just to add some spacing between radio options. Thanks!!

**Comment by chris-at-jobber** [comment]
@tyler-santos I don't see any difference between your before and after images, are those accurate?

File: packages/components/src/RadioGroup/RadioGroup.css [hunk 1]
```diff
@@ -12,6 +12,7 @@
 .label {
   display: inline-flex;
   align-items: center;
+  padding-bottom: 0.5rem;
```
Comment (seanhealy) [inline packages/components/src/RadioGroup/RadioGroup.css:15]: This should use one of our spacing units.

https://atlantis.getjobber.com/space

File: packages/components/src/RadioGroup/RadioGroup.css [hunk 2]
```diff
@@ -38,3 +39,7 @@
   display: inline-flex;
   flex-direction: column;
 }
+/* This to override the styling being applied by playground. */
```
Comment (JeffMarvin) [inline packages/components/src/RadioGroup/RadioGroup.css:42]: What's this for?

**Comment by chris-at-jobber** [comment]
This PR is no longer required!

**Pull request closed without merging**